pandas/tests/generic/test_series.py

```diff
@@ -240,17 +243,9 @@ def test_to_xarray_index_types(self, index):
         tm.assert_series_equal(result.to_series(), s, check_index_type=False)
 
     @td.skip_if_no("xarray", min_version="0.7.0")
-    def test_to_xarray(self):
+    def test_to_xarray_multiindex(self):
```

Comment:
Potentially this could benefit from using the MultiIndex cases from `indices`. I didn't want to include it in this PR though